geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/LauncherLifecycleCommands.java

```diff
@@ -771,11 +771,18 @@ public Result statusLocator(
               CliStrings.STATUS_SERVICE__GFSH_NOT_CONNECTED_ERROR_MESSAGE, LOCATOR_TERM_NAME));
         }
       } else {
-        final LocatorLauncher locatorLauncher =
-            new LocatorLauncher.Builder().setCommand(LocatorLauncher.Command.STATUS)
-                .setBindAddress(locatorHost).setDebug(isDebugging()).setPid(pid)
-                .setPort(locatorPort).setWorkingDirectory(workingDirectory).build();
+        final LocatorLauncher locatorLauncher;
 
+        if ((locatorHost == null) && (locatorPort == null) && (workingDirectory == null)) {
+          return ResultBuilder.createShellClientErrorResult(
+              String.format(CliStrings.STATUS_LOCATOR__NO_LOCATOR_SPECIFIED_ERROR_MESSAGE,
+                  getLocatorId(locatorHost, locatorPort),
+                  StringUtils.defaultIfBlank(workingDirectory, SystemUtils.CURRENT_DIRECTORY)));
```

Comment:
workingDirectory == null in this clause of the if, so this argument can just be SystemUtils.CURRENT_DIRECTORY